wp-content/plugins/core/src/Core.php

```diff
@@ -100,23 +93,43 @@ private function load_service_providers() {
 		}
 	}
 
+
+	/**
+	 * Register optional dependencies if they exist. Override
+	 * the service provider for the dependency to change its
+	 * behavior.
+	 */
+	private function optional_dependencies() {
+		$optional_dependencies = [
+			'blog_copier'  => '\Tribe\Libs\Blog_Copier\Blog_Copier_Provider',
+			'container'    => '\Tribe\Libs\Container\Container_Provider',
+			'generators'   => '\Tribe\Libs\Generators\Generator_Provider',
+			'queues'       => '\Tribe\Libs\Queues\Queues_Provider',
+			'queues-mysql' => '\Tribe\Libs\Queues_Mysql\Mysql_Backend_Provider',
+		];
+
+		foreach ( $optional_dependencies as $key => $provider ) {
+			$this->providers[ $key ] = new $provider();
```

Comment:
```suggestion
			if( class_exists( $provider ) ) { 
				$this->providers[ $key ] = new $provider();
			}
```

Reply:
I ran `./composer.sh remove moderntribe/square1-blog-copier` as a test and threw a 500. If these are optional, we should definitely just check if they exist before attempting to load them.

Reply:
> If these are optional, we should definitely just check if they exist before attempting to load them

✅ done